src/components/select/select.js

```diff
@@ -1250,7 +1250,15 @@ function SelectProvider($$interimElementProvider) {
         element
           .removeClass('md-active')
           .attr('aria-hidden', 'true')
-          .css('display', 'none');
+          .css({
+            'display': 'none',
+            'top': '',
+            'right': '',
+            'bottom': '',
+            'left': '',
+            'font-size': '',
+            'min-width': ''
```

Comment:
Thankfully it looks like it results in these styles being cleared rather than having them set to empty strings.